### DISCUSSION TITLE
Unable to catch the error thrown by dev Server on port conflict.

### DISCUSSION THREAD

**Discussion by subodhkumar** [webpack/webpack#16669] [Q&A]
We have a wrapper on top of `webpack-dev-server`. 
With webpack 4, when we started dev-server passing a port which was occupied we were getting `EDDRINUSE` logged in the console.
However, with webpack5 we do not see it getting logged & unable to catch the [error thrown](https://github.com/webpack/webpack-dev-server/blob/d1d14340d34ed5e5a97293cee09f49a85428c46f/lib/Server.js#L2512) by the server in our wrapper code.

Any suggestion on how to handle it & catch the error emitted.

Thanks in advance

**Comment by alexander-akait** [comment]
Webpack dev server by defau;t search a free port, so why you need to catch this, also how do you try to catch?
